src/constructs/lambda/lambda.ts

```diff
@@ -12,7 +12,7 @@ import type { GuStack } from "../core";
 import { AppIdentity, GuDistributionBucketParameter } from "../core";
 import { ReadParametersByName, ReadParametersByPath } from "../iam";
 
-export interface GuFunctionProps extends GuDistributable, Omit<FunctionProps, "code">, AppIdentity {
+export interface GuFunctionProps extends GuDistributable, Omit<FunctionProps, "code" | "logFormat">, AppIdentity {
```

Comment:
Do we mean to remove the `logFormat` property?

Reply:
I did mean to remove it, but I think you're right, we don't actually need to do this.